src/draw/lv_image_buf.h

```diff
@@ -92,6 +92,7 @@ typedef struct {
     uint32_t flags: 16;         /*Image flags, see `lv_image_flags_t`*/
     uint32_t cf : 8;            /*Color format: See `lv_color_format_t`*/
     uint32_t magic: 8;          /*Magic number. Must be LV_IMAGE_HEADER_MAGIC*/
+    uint32_t rotation : 16;     /*Image angle of rotation*/
```

Comment:
I think the source image's rotation is not relevant for LVGL drawing feature, so it should not be stored in LVGL image buf, because we don't process that meta info. That is only a meta info.

-> The decoder should decode the image in the correct angle, so LVGL can show it correctly.

Reply:
Okay, I will fix it.